confgenerator/confgenerator.go

```diff
@@ -101,11 +112,8 @@ func (uc *UnifiedConfig) GenerateOtelConfig(ctx context.Context) (string, error)
 		ReceiverPipelineName: "fluentbit",
 	}
 
-	if uc.Metrics.Service.LogLevel == "" {
```

Comment:
Your current diff will cause the logging log level to override the metrics log level. I think we should stick with the existing behavior of only using the metrics log level, which you could do with

```go
logLevel := "info"
if uc.Metrics != nil && uc.Metrics.Service != nil && uc.Metrics.Service.LogLevel != "" {
  logLevel = uc.Metrics.Service.LogLevel
}
```

Reply:
Done.